packages/dbml-cli/__test__/db2dbml/mssql/expect-out-files/schema.dbml

```diff
@@ -13,55 +13,70 @@ Enum "dbo"."chk_gender" {
 }
 
 Table "dbo"."Authors" {
-  "AuthorID" int [pk, not null]
-  "AuthorName" nvarchar [unique]
-  "BirthYear" int [unique]
-  "NationalityID" int [pk, not null]
+  "AuthorID" int(10) [pk, not null]
+  "AuthorName" nvarchar(100) [unique]
+  "BirthYear" int(10) [unique]
+  "NationalityID" int(10) [pk, not null]
 }
 
 Table "dbo"."Books" {
-  "AuthorID" int [pk, not null]
-  "BookID" int [pk, not null]
-  "ISBN" nvarchar [unique]
-  "NationalityID" int
-  "Title" nvarchar
+  "AuthorID" int(10) [pk, not null]
+  "BookID" int(10) [pk, not null]
+  "ISBN" nvarchar(20) [unique]
+  "NationalityID" int(10)
+  "Title" nvarchar(200)
 }
 
 Table "dbo"."DatetimeTypes" {
   "DATECol" date [default: `getdate()`]
-  "DATETIME2Col" datetime2 [default: `sysdatetime()`]
+  "DATETIME2Col" datetime2(7) [default: `sysdatetime()`]
```

Comment:
We will not want to show the users the length of datetime types. Please help me to fix it nha.

Reply:
I reverted to this [commit](https://github.com/holistics/dbml/pull/645/commits/2310156d58c53e8fcf57974b58db9ecf69f9450d) nha a